pkg/security/authentication.go

```diff
@@ -25,8 +25,11 @@ import (
 	"istio.io/istio/pkg/env"
 )
 
+// AuthPlaintext allows non TLS connections, for cases when Istiod is collocated with a gateway or on a secure
+// network (like ambient). For Istio CA it also requires disabling the TLS port on 15012, which will activate
```

Comment:
Could you clarify this comment? What CA configuration is Istio CA? Not sure exactly what configuration you're referring to. Could you also link to where the TLS port is disabled? Thanks!

Reply:
I meant 'citadel grpc' - that's the Istio CA interface. I'll try to find the setting, I'm sure it's somewhere :-), the comment is based on looking at the code that registers gRPC interfaces on the 15010 port - the citadel (signing) gRPC is registered only if 15012 is disabled.